Makefile

```diff
@@ -135,6 +135,10 @@ vm: $(VM_IMAGE)
 check: $(VM_IMAGE)
 	PYTHONPATH=`pwd`/bots/machine test/check-application -v -b $(BROWSER) -C 2 -M 2048 $(TEST_OS)
 
+# debug end to end test
+debug-test: $(VM_IMAGE)
+	DEBUG_TEST=true PYTHONPATH=`pwd`/bots/machine test/check-application -v -b $(BROWSER) -C 2 -M 2048 $(TEST_OS)
```

Comment:
how about calling `DEBUG_TEST=true $(MAKE) check` to avoid duplicating the actual invocation? And maybe rename this to `debug-check` for consistency with `check`?